orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -960,6 +960,10 @@ def __init__(self):
         self.preview = ""
         gui.label(box, self, "%(preview)s", wordWrap=True)
         self.controlArea.layout().insertWidget(1, box)
+        box = gui.vBox(self.controlArea, "Output")
```

Comment:
I think we once agreed to put the input-unrelated info into the buttonsArea.

![image](https://user-images.githubusercontent.com/11465003/121854819-f2a53f80-ccf2-11eb-8724-dc223c49bef7.png)

If you prefer it your way, feel free to do it like that.
If not, here are some hints:
- set `buttons_area_orientation = Qt.Vertical`
- replace `box = gui.vBox(self.controlArea, "Output")` with `box = gui.vBox(self.buttonsArea, "Output")`
- replace `self.controlArea.layout().insertWidget(2, box)` with `self.buttonsArea.layout().insertWidget(0, box)`